grizzly/users/base/grizzly_user.py

```diff
@@ -68,14 +69,16 @@ def stop(self, force: bool = False) -> bool:
     def request(self, request: RequestTask) -> GrizzlyResponse:
         raise NotImplementedError(f'{self.__class__.__name__} has not implemented request')
 
-    def render(self, request: RequestTask) -> Tuple[str, str, Optional[str]]:
+    def render(self, request: RequestTask) -> Tuple[str, str, Optional[str], Optional[Dict[str, str]], Optional[Dict[str, str]]]:
```

Comment:
I feel like `render` should maybe be moved to `RequestTask` now, since it has gotten a bit big. However, it's a bit out-of-scope for this PR, consider this comment as an mental note for my self :)